crates/rolldown_error/src/build_error/mod.rs

```diff
@@ -120,6 +120,12 @@ impl From<anyhow::Error> for BuildDiagnostic {
   }
 }
 
+impl From<oxc_resolver::ResolveError> for BatchedBuildDiagnostic {
+  fn from(err: oxc_resolver::ResolveError) -> Self {
+    Self::new(vec![BuildDiagnostic::unhandleable_error(err.into())])
```

Comment:
cc @hyf0 The `ResolveError` look like needs to give a specific error.

Reply:
We have `DiagnosableResolveError` for it,  but its applicability was too limited.

Reply:
Yeah. It could be improved.